public/playersBetController.js

```diff
@@ -0,0 +1,147 @@
+"use strict";
+export default class playersBetController {
+  constructor(players) {
+    this.currentPlayerTurn = 0;
+    this.players = players;
+    this.activePlayer = this.players[this.currentPlayerTurn];
+    this.currentBet = 0;
+    this.playersStillBetting = false;
+
+    const controlsContainerEle = document.querySelector("#game");
+    this.controlsContainerEle = controlsContainerEle;
+  }
+
+  //Cada jugador debe hacer su apuesta antes de la reparticion de cartas
+
+  //Despues de repartir las cartas sila carta del dealer es As, los jugadores pueden hacer una apuesta de seguro, el maximo que se puee apostar es la miad de la apuesta inicial , se deja en la llinea de apuesta segura y se paga 2 a 1
+  // si se hace apuesta de seguro luego el jugador inicia su mano con normalidad, es decir con las mismas opciones de siempre
+
+  controlsHtml() {
+    return `
+    <div class='bet-controls'>
+        <div class='bet-controls__pot'>Pot: ${this.activePlayer.pot}</div>
+        <div class='bet-controls__bet'>Current bet: ${this.currentBet}</div>    
+        <button type='button' class= 'bet-controls__btn btn--25'>
+        25
+        </button>
+        <button type='button' class= 'bet-controls__btn btn--50'>
+        50
+        </button>
+        <button type='button' class= 'bet-controls__btn btn--100'>
+        100
+        </button>    
+        <button type='button' disabled class= 'bet-controls__btn btn--clear'>
+        Clear
+        </button>
+        <button type='button' disabled class= 'bet-controls__btn btn--bet'>
+        Confirm bet
+        </button>
+    </div>    
+      `;
+  }
+
+  rederBetControls() {
+    this.controlsContainerEle.insertAdjacentHTML(
+      "beforeend",
+      this.controlsHtml()
+    );
+  }
+
+  initPlayers() {
+    this.amountOfPlayers = this.players.length;
```

Comment:
It doesn't make much sense to store the amount of player because we always have access to the list of players, so it is best to directly call `this.players.length` whenever the amount of players is needed.